UICatalog/Scenario.cs

```diff
@@ -149,7 +149,7 @@ public static ObservableCollection<Scenario> GetScenarios ()
     public virtual void Main () { }
 
     private const uint BENCHMARK_MAX_NATURAL_ITERATIONS = 500; // not including needed for demo keys
-    private const int BENCHMARK_KEY_PACING = 1; // Must be non-zero
+    private const int BENCHMARK_KEY_PACING = 10; // Must be non-zero
```

Comment:
I think more robust fix is to look into the runstate changes that's actually happening that triggers the addition of a new timeout set of demo keys. E.g. add a flag or collection recording which demo keys have already been sent (`HashSet<Scenario>`, for example)

See also this comment https://github.com/gui-cs/Terminal.Gui/issues/3941#issuecomment-2692502887

Reply:
Agree. I'll merge this now and we can then do it right via #3949